assets/product-images-slider.js

```diff
@@ -0,0 +1,62 @@
+const thumbnailSlides = document.querySelector('[data-glide-thumbnail-slides]').dataset.glideThumbnailSlides
+const thumbnailsPerView = thumbnailSlides ? thumbnailSlides : 0
+
+const glideOptions = {
+  image: {
+    type: 'carousel',
+    startAt: 0,
+    perView: 1,
+    focusAt: 'center',
+    gap: 0,
+    dragThreshold: 10,
+    peek: 40,
+  },
+  thumbnail: {
+    type: 'carousel',
+    startAt: 0,
+    perView: thumbnailsPerView,
+    focusAt: thumbnailsPerView % 2 === 0 ? (thumbnailsPerView / 2) - 1 : 'center'
```

Comment:
I'd reccomend putting the result of this conditional into a variable, so it's easier to read the intention